lessons/beginners/fstring/info.yml

```
@@ -0,0 +1,5 @@
+title: Šablony (formátovací řetězce)
+style: md
+attribution:
+- Pro PyLadies Brno napsal Petr Viktorin, 2018-2019.
+license: cc-by-sa-40
```

Is it possible to use CC0 for new code snippets?

```suggestion
license: cc-by-sa-40
license_code: cc0
```

---

Yes. I should go through the old ones, too.